src/container/srv_container.c

```diff
@@ -4205,3 +4205,85 @@ ds_cont_get_prop(uuid_t pool_uuid, uuid_t cont_uuid, daos_prop_t **prop_out)
 	cont_svc_put_leader(svc);
 	return rc;
 }
+
+/*
+ * Check whether the specified container exists in the container service or not.
+ * If yes, return the container label via the @prop.
+ */
+int
+ds_cont_existence_check(struct cont_svc *svc, uuid_t uuid, daos_prop_t **prop)
+{
+	struct cont	*cont = NULL;
+	daos_prop_t	*tmp = NULL;
+	struct rdb_tx	 tx;
+	int		 rc;
+
+	rc = rdb_tx_begin(svc->cs_rsvc->s_db, svc->cs_rsvc->s_term, &tx);
+	if (rc != 0)
+		goto out;
+
+	ABT_rwlock_rdlock(svc->cs_lock);
+	rc = cont_lookup(&tx, svc, uuid, &cont);
+	if (rc != 0)
+		goto out_tx;
+
+	rc = cont_prop_read(&tx, cont, DAOS_CO_QUERY_PROP_LABEL, &tmp, true);
+	if (rc != 0)
+		D_GOTO(out_cont, rc = (rc == -DER_NONEXIST ? 0 : rc));
```

Comment:
[question]There is DER_NONEXIST check in the caller? why return 0 here, a bit confusing

Reply:
Ah, i see, this is treated as normal and skipped.

Reply:
If the ds_cont_existence_check() returns -DER_NONEXIST, it will be regarded as that the container itself does not exist in the container server. But here, it is the "label" property does not exist. So we convert it as zero before return to avoid misguiding. On the other hand, because we do not set "*prop" output parameter, then the caller will know that the container does not have "label" property.